tests/Meziantou.Analyzer.Test/Rules/ClassWithEqualsTShouldImplementIEquatableTAnalyzerTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Threading.Tasks;
```

Comment:
You only test classes and interfaces. You should also test structs.

Reply:
Done